mayastor/src/subsys/nvmf/mod.rs

```diff
@@ -83,7 +84,10 @@ impl Nvmf {
     extern "C" fn init() {
         debug!("mayastor nvmf subsystem init");
 
-        // this code only ever gets run on the fist core
+        // this code only ever gets run on the first core
+
+        // set up custom NVMe Admin command handler
+        nexus_bdev::setup_create_snapshot_hdlr();
```

Comment:
right, i would expect all the nvmf related code in the nexus_bdev to be placed here as well. (in a seperate file)